includes/templates/reader-activation/login-form.php

```diff
@@ -0,0 +1,10 @@
+<?php
+/**
+ * Reader Activation login form.
+ * This is an empty file in order to trick WooCommerce's My Account login template
+ * into rendering nothing so that we can replace it with our own login form.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
```

Comment:
```suggestion
namespace Newspack;

Reader_Activation::render_auth_form( true );
```

Instead of rendering nothing, the method could be called here

Reply:
Great idea, that's much less hacky feeling! 4d6f9d2